src/cli/cli.cpp

```diff
@@ -83,6 +84,10 @@ Ip6::IcmpEcho Interpreter::sIcmpEcho(&HandleEchoResponse, NULL);
 Ip6::SockAddr Interpreter::sSockAddr;
 Server *Interpreter::sServer;
 uint8_t Interpreter::sEchoRequest[1500];
+long Interpreter::sLength = 8;
+long Interpreter::sCount = 1;
+long Interpreter::sInterval = 1000;
```

Comment:
@jwhui , yes, agree. Do you think I also need to change other long variables to uint8_t, uint16_t, or uint32_t, like channel, child time out, reuse delay, etc?